labonneboite/importer/jobs/geocode.py

```diff
@@ -175,19 +175,25 @@ def validate_coordinates(self):
         con.close()
 
     @timeit
-    def run_geocoding_jobs(self, geocoding_jobs):
+    def run_geocoding_jobs(self, geocoding_jobs, multithreading_disabled=False):
         adresses_not_geolocated[:] = []
         coordinates_updates[:] = []
-        pool = Pool(processes=pool_size)
+
         logger.info("Nombre de geocoding jobs : {}".format(
             len(geocoding_jobs)))
-        for siret, full_address, initial_coordinates, city_code in geocoding_jobs:
-            pool.apply_async(self.find_coordinates_for_address,
-                             (siret, full_address, initial_coordinates, city_code,))
-        # for siret, full_address, initial_coordinates, city_code in geocoding_jobs:
-        #    self.find_coordinates_for_address(siret,full_address,initial_coordinates,city_code)
-        pool.close()
-        pool.join()
+
+        if multithreading_disabled is False:
```

Comment:
perso j'aurai choisi l'autre ordre (True puis False) comme ça même pas besoin de `is True/False` et plus lisible. Pinaillage.